ui/components/ui/icon/icon-block-explorer.js

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+/**
```

Comment:
I think this change is what is causing the merge conflict. `icon-blocker-explorer` has been removed. If you remove these changes and rebase from `develop` you should be good!

Reply:
@georgewrmarshall, I have removed the changes from `icon-blocker-explorer`